libsupport/include/katana/EntityTypeManager.h

```diff
@@ -17,10 +17,16 @@
 
 namespace katana {
 
-/// EntityTypeID uniquely identifies an entity (node or edge) type
-/// EntityTypeID for nodes is distinct from EntityTypeID for edges
-/// This type may either be an atomic type or an intersection of atomic types
-/// EntityTypeID is represented using 8 bits
+/// EntityTypeID uniquely identifies an entity (node or edge) type, which is
+/// a Boolean property.
```

Comment:
A type is a boolean property? This statement is confusing to me; why isn't `EntityTypeID` a bool then?

Reply:
The term property is way overloaded, I should change that.  Any suggestions?
A type is a Boolean though, right?

Reply:
I thought a type was a set of labels?

Reply:
What is a label?

Reply:
One of these strings in the sets we keep talking about.

Reply:
Ok, see if you like what I have now.

Reply:
Molto bene! tyty.